README.md

```diff
@@ -49,6 +49,8 @@
 
    cdes_metadata_path = "./tests/demo_data"
 
+   algorithm_folders = "./mipengine/algorithms,./tests/algorithms"
```

Comment:
it would maybe make sense to have two separate variables for this
```
algorithm_folder = "./mipengine/algorithms"
tset_algorithm_folder="./tests/algorithms"
```
cause, first this should be a array/list and it is not, and second it clearly differentiates what's what

Reply:
The node/controller are reading these values from an env variable. It has to be one variable and to be a string in order for it to work.

Reply:
I don't think it's a good idea to hardcode logic such as a "test_algorithm_folder". What if in the future we need 3 folders?

A more generic and dynamic approach is better in my opinion.

Reply:
ok let's leave it like that and see how it goes